documentation/source/coroutines.rst

```diff
@@ -144,13 +141,16 @@ forcing their completion before they would naturally end.
 .. versionchanged:: 1.4
     The :any:`cocotb.coroutine` decorator is no longer necessary for ``async def`` coroutines.
     ``async def`` coroutines can be used, without the ``@cocotb.coroutine`` decorator, wherever decorated coroutines are accepted,
-    including :keyword:`yield` statements and :func:`cocotb.fork`.
+    including :keyword:`yield` statements and :func:`~cocotb.start` and :func:`~cocotb.start_soon`.
```

Comment:
Isn't this rewriting history?
Maybe better as
```suggestion
    including :keyword:`yield` statements and ``cocotb.fork`` (since replaced with :func:`~cocotb.start` and :func:`~cocotb.start_soon`).
```
(if at all).